packages/data-stores/src/launch/actions.ts

```diff
@@ -52,11 +55,27 @@ export const setDomainSearch = ( domainSearch: string ) =>
 		domainSearch,
 	} as const );
 
-export const setPlanProductId = ( planProductId: number | undefined ) =>
+const __setBillingPeriod = ( billingPeriod: Plans.PlanBillingPeriod ) =>
```

Comment:
No need of prefixing with `__`, as this function is not exported anyway (plus, we may actually need it when working on #49253)